service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantMidEventControllerTest.java

```diff
@@ -0,0 +1,250 @@
+package uk.gov.hmcts.reform.fpl.controllers;
```

Comment:
I would like to see assertions around PBA number in the tests here as well.

Reply:
👍